lib/Importer/ONNX.cpp

```diff
@@ -68,16 +68,24 @@ bool ONNXModelLoader::loadProtoFile(onnx::GraphProto &net,
   return true;
 }
 
-size_t getPad(ArgumentDictionaryTy &dict) {
+std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
   if (dict.count("pads")) {
-    return getConstantArrayHead(dict["pads"]);
-  } else if (dict.count("auto_pad")) {
+    return getShape(dict["pads"]);
+  } 
+  if (dict.count("auto_pad")) {
     auto padStr = loadStr(dict["auto_pad"]);
-    if (padStr == "VALID")
-      return 0;
+    if (padStr == "VALID") {
+      // Return default value 0 for pad.
+      return {0, 0, 0, 0};
+    }
     assert(false && "only auto_pad==VALID is supported");
   }
-  return 0;
+  // Return default value 0 for pad.
+  return {0, 0, 0, 0};
+}
+
+size_t getPad(ArgumentDictionaryTy &dict) {
```

Comment:
ditto

Reply:
I reverted the original getPad() function. I just think of that the original getPad() requires that all pads are equal, otherwise there is an assertion. But getPads() doesn't has such requirement.